src/Model/PartnerManager.php

```diff
@@ -39,4 +39,14 @@ public function update(array $partner): bool
 
         return $statement->execute();
     }
+
+    public function showPartnerWine(int $id): array
```

Comment:
This is a bad name for this function, it should somtehing like "getWinesByPartner"